### ISSUE TITLE
[BUG] The kernel appears to have died while running AutoGluon in Jupyter

### ISSUE THREAD

**Issue by c3-varun** [autogluon/autogluon#2078]
- [ X] I have checked that this bug exists on the latest stable version of AutoGluon
- [ ] and/or I have checked that this bug exists on the latest mainline of AutoGluon via source installation

**Describe the bug**
On both 5.1 and 5.2 my Jupyter kernel dies when running AutoGluon on a very small and simple dataset. I tried changing the time_limit to 10, 30, 60 and got the same result in all attempts.

**Expected behavior**
I expect the training to run without error.

**To Reproduce**
Please see attached minimal Jupyter notebook (delete .txt from the name and run it as a ipynb).

**Screenshots**
<img width="3008" alt="image" src="https://user-images.githubusercontent.com/39346102/186772406-bb3215e7-358d-4210-a62b-b4c37d4e0f57.png">

**Installed Versions**
Which version of AutoGluon are you are using?  I have tried both 5.1 and 5.2 with the same result
INSTALLED VERSIONS
------------------
date                   : 2022-08-25
time                   : 14:36:03.588735
python                 : 3.9.7.final.0
OS                     : Darwin
OS-release             : 21.6.0
Version                : Darwin Kernel Version 21.6.0: Wed Aug 10 14:25:27 PDT 2022; root:xnu-8020.141.5~2/RELEASE_X86_64
machine                : x86_64
processor              : i386
num_cores              : 16
cpu_ram_mb             : 65536
cuda version           : None
num_gpus               : 0
gpu_ram_mb             : []
avail_disk_size_mb     : 683704

autogluon.common       : 0.5.1
autogluon.core         : 0.5.1
autogluon.features     : 0.5.1
autogluon.multimodal   : 0.5.1
autogluon.tabular      : 0.5.1
autogluon.text         : 0.5.1
autogluon.timeseries   : None
autogluon.vision       : 0.5.1
autogluon_contrib_nlp  : None
boto3                  : 1.24.59
catboost               : 1.0.6
dask                   : 2021.10.0
distributed            : 2021.10.0
fairscale              : 0.4.6
fastai                 : 2.7.9
gluoncv                : 0.11.0
hyperopt               : 0.2.7
lightgbm               : 3.3.2
matplotlib             : 3.4.3
networkx               : 2.6.3
nlpaug                 : 1.1.10
nltk                   : 3.6.5
nptyping               : 1.4.4
numpy                  : 1.21.6
omegaconf              : 2.1.2
pandas                 : 1.3.4
PIL                    : 9.0.1
protobuf               : None
psutil                 : 5.8.0
pytorch-metric-learning: None
pytorch_lightning      : 1.6.5
ray                    : 1.13.0
requests               : 2.26.0
scipy                  : 1.7.1
sentencepiece          : None
skimage                : 0.19.3
sklearn                : 1.0.2
smart_open             : 5.2.1
timm                   : 0.5.4
torch                  : 1.12.1
torchmetrics           : 0.7.3
torchtext              : 0.13.1
torchvision            : 0.13.1
tqdm                   : 4.62.3
transformers           : 4.20.1
xgboost                : 1.4.2
​

```python
# Replace this code with the output of the following:
from autogluon.core.utils import show_versions
show_versions()
```

</details>

**Additional context**
I have a 64GB RAM MacBook Pro that had only 22GB used (42GB available) when running this notebook.

[TestAutoGluon.ipynb.txt](https://github.com/awslabs/autogluon/files/9428380/TestAutoGluon.ipynb.txt)
[training_data.csv](https://github.com/awslabs/autogluon/files/9428381/training_data.csv)

**Comment by Chriluo** [comment]
same issue, but I successfully ran the script in Pycharm.

**Comment by lovechang1986** [comment]
me too

**Comment by gradientsky** [comment]
I would suggest to try a few things (it runs without issues on my box):
1) make a clean virtual environment
2) install the most recent version of AutoGluon

**Comment by Innixma** [comment]
Marking as resolved for now. Please try with latest pre-release. If issue persists, please re-open the issue and provide details on the exact part of the training process that the kernel dies.
